hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -372,18 +388,14 @@ void notifyError(Object error) {
     }
 
     void notifyNormalResponse(Object value, int expectedBackups) {
-        if (value == null) {
-            value = NULL_RESPONSE;
-        }
-
-        //if a regular response came and there are backups, we need to wait for the backs.
-        //when the backups complete, the response will be send by the last backup or backup-timeout-handle mechanism kicks on
+        // if a regular response came and there are backups, we need to wait for the backs.
+        // when the backups complete, the response will be send by the last backup or backup-timeout-handle mechanism kicks on
 
         if (expectedBackups > backupsCompleted) {
             // So the invocation has backups and since not all backups have completed, we need to wait.
             // It could be that backups arrive earlier than the response.
 
-            this.pendingResponseReceivedMillis = Clock.currentTimeMillis();
+            this.pendingResponseReceivedMillis = currentTimeMillis();
```

Comment:
I would not use a static import of `System.currentTimeMillis()` here. We also have `Clock.currentTimeMillis()`, so you don't know which clock is used here.

Since you also used `Clock.currentTimeMillis()` in this class: Are you sure the correct clock was used in both places? I have no clue when to use which one, it just looks suspicious to have them both so close together.